### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by litzenberger**
Adding usage and test descriptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Run individual test
+    TODO:
+
+##  Results
+
+  Results will be writen to out directory unders tests
+
+## See results in Grafana
+  
+  Coming soon....
```
Comment (mcollina) [inline README.md]: @Elexy can you fill this in after this lands?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Test description
+
+    ### local-add:
```
Comment (mcollina) [inline README.md]: you should move this to the top of the line, no indentation.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Local Setup
+
+    Clone Benchmark tests:
+
+    git clone https://github.com/ipfs/benchmarks.git
+
+   cd ../benchmarks/tests
+
+   npm install
```
Comment (mcollina) [inline README.md]: this would need to be put within ``  blocks.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   
+## Run tests locally
+
+    From the benchmark/tests directory:
+      - node local-add
+      - node loca-extract
+      - node local-transfer
+
+    Run all benchamrks:
+      - npm run benchmark
```
Comment (mcollina) [inline README.md]: remove indentation and use code blocks.

File: README.md (hunk 1 above)
Comment (litzenberger) [reply README.md]: Ill add this now since it is in master.

Comment (Elexy) [reply README.md]: I can PR again with any corrections tomorrow

**Pull request merged**